markdown/ch1.markdown

```diff
@@ -883,15 +883,53 @@ N = np.sum(counts, axis=0)  # sum each column to get total reads per sample
 L = gene_lengths  # lengths for each gene, matching rows in `C`
 ```
 
-First, we multiply by 10^9.
+> **Tip: Numbers and computers {.callout}**
+>
+> We can't cover everything you need to know about numeric representations
+> in computers in just a tip box, but you *should* know that numbers are
+> represented as "n-bit precision" "integer" or "floating point" numbers in
```

Comment:
I don't think the term "precision" is typically used with integers.  All integer representations are equally precise.